deployment/windows/confirmation.nsdinc

```diff
@@ -0,0 +1,52 @@
+; =========================================================
+; This file was generated by NSISDialogDesigner 1.4.4.0
+; http://coolsoft.altervista.org/nsisdialogdesigner
+;
+; Do not edit it manually, use NSISDialogDesigner instead!
+; Modified by EraYaN (2019-08-31)
+; =========================================================
+
+; handle variables
+Var hCtl_confirmation
+Var hCtl_confirmation_ConfirmRichText
+
+!include StrSlash.nsh
+
+; dialog create function
+Function fnc_confirmation_Create
+  ; === confirmation (type: Dialog) ===
+  nsDialogs::Create 1018 ; THIS IS THE LINE THAT CRASHES THE INSTALLER
```

Comment:
This is the line that crashes the built installer. But only if the service account type page was shown. Probably something not cleaned up properly.

Reply:
Crashes with this in the debugger:

```
Exception thrown at 0x77565DAE (user32.dll) in jellyfin_10.3.7_windows-x64.exe: 0xC00000FD: Stack overflow (parameters: 0x00000001, 0x000A2FBC).
Unhandled exception at 0x77565DAE (user32.dll) in jellyfin_10.3.7_windows-x64.exe: 0xC000041D: An unhandled exception was encountered during a user callback.
```